packages/block-library/src/block/edit.js

```diff
@@ -127,6 +165,18 @@ export default function ReusableBlockEdit( { attributes: { ref }, clientId } ) {
 							showTooltip
 						/>
 					</ToolbarGroup>
+					<ToolbarGroup>
+						<ToolbarButton
+							isPrimary
+							className="block-library-block__reusable-block-save-button"
+							onClick={ saveEditedRecords }
+							label={ __( 'Save globally' ) }
```

Comment:
Might need some copy review on this label. Perhaps it could be 'Save reusable block'.